### ISSUE TITLE
KOTOR/KOTOR2: Missing heads on several creature models

### ISSUE THREAD

**Issue by DrMcCoy** [xoreos/xoreos#240]
Several NPCs are missing heads. For example here in the module tar_m02ab.
![20180510t111446](https://user-images.githubusercontent.com/156130/39867238-43137a4c-5454-11e8-9d2a-e635c097cdcf.png)

One draw of the rng in the character generator of KotOR2 even created a headless PC for me, though that might be a bug in the generator.

**Comment by DrMcCoy** [comment]
Ah, got it, it's this one: 
![20180510t113059](https://user-images.githubusercontent.com/156130/39867712-7e93ba58-5456-11e8-8b3e-b7545c5a4be6.png)

```
WARNING: Failed to create texture "PFHC02" (-1)
    Because: No such image resource "PFHC02"
WARNING: Failed to create texture "PFHC02" (-1)
    Because: No such image resource "PFHC02"
WARNING: Failed to create texture "w_LghtSbr" (-1)
    Because: No such image resource "w_LghtSbr"
WARNING: Failed to create texture "w_LghtSbr" (-1)
    Because: No such image resource "w_LghtSbr"
WARNING: Failed to create texture "PFHC02" (-1)
    Because: No such image resource "PFHC02"
WARNING: Duplicate animation "f3p1a" in model "S_Female01"!
WARNING: Position controller with 9 values!
WARNING: Failed to load object model "pfhd01"
    Because: No such MDL "pfhd01"
```

**Comment by DrMcCoy** [comment]
No, GitHub, that issue was supposed to stay open, even though the commit said "Fix #240".

**Comment by seedhartha** [comment]
Partially (?) fixed with #292.

**Comment by DrMcCoy** [comment]
Right, yeah, that seems to be fixed :)
